ScoutSuite/providers/aws/services/rds.py

```diff
@@ -66,19 +64,21 @@ def parse_snapshot(self, global_params, region, dbs):
         for attribute in attributes:
             snapshot[attribute] = dbs[attribute] if attribute in dbs else None
         api_client = api_clients[region]
-        attributes = api_client.describe_db_snapshot_attributes(DBSnapshotIdentifier = snapshot_id)['DBSnapshotAttributesResult']
+        attributes = api_client.describe_db_snapshot_attributes(DBSnapshotIdentifier=snapshot_id)[
+            'DBSnapshotAttributesResult']
```

Comment:
Do we really want to enforce the maximum line length all the time? I'm not a big fan of splitting a line like this.

Reply:
PEP8 ;)

Reply:
One of the high points of respecting coding styles is to prevent conflicts. If everyone follows the coding styles, git won't flag a bunch of conflicts where the same line is changed the same way, but with different spacing and stuff